app/src/main/java/com/epfl/drawyourpath/mainpage/fragments/ChatFragment.kt

```diff
@@ -0,0 +1,53 @@
+package com.epfl.drawyourpath.mainpage.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.epfl.drawyourpath.R
+import com.epfl.drawyourpath.chat.Chat
+import com.epfl.drawyourpath.mainpage.fragments.helperClasses.ChatAdapter
+
+class ChatFragment : Fragment() {
+
+    private lateinit var chatRecyclerView: RecyclerView
+    private lateinit var chatAdapter: ChatAdapter
+    private val chatList = mutableListOf<Chat>()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View? {
+        return inflater.inflate(R.layout.fragment_chat_list, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        chatRecyclerView = view.findViewById(R.id.chatListRecyclerView)
+        chatRecyclerView.layoutManager = LinearLayoutManager(requireContext())
+
+        // Add mock chats to the chatList
+        for (i in 1..10) {
+            val chat = Chat()
+            chatList.add(chat)
+        }

```

Comment:
We have a fixed number of chats but we should be able to create new chats, that would be a good issue to do later.

Reply:
This is just to showcase the data for the UI. It will be changed when we fetch the real data in the future